styles.css

```diff
@@ -0,0 +1,571 @@
+  border-bottom: 1px solid #f42;
+}
+
+.lighter {
+  background: #f8eae7;
+}
+
+.labels i {
+  padding-right: 0.8rem;
+}
+
+.label2,
+.label3 {
+  display: block;
+  padding: 1rem 0 0 1rem;
+  color: #333;
+}
+
+.label3 {
+  color: rgb(165, 161, 161);
+  padding-bottom: 1rem;
+}
+
+.company img {
+  position: absolute;
+  width: 60px;
+  height: 60px;
+  margin: 10px 10px;
+}
+
+.bold {
+  font-weight: bold;
+}
+
+.company-holder {
+  display: grid;
+  grid-gap: 1rem;
+  grid-template-rows: 1fr;
+  margin-top: 15px;
+}
+
+.txt-line {
+  border-top: 0.1px solid #f42;
+  width: 90%;
+  margin: 0 auto;
+}
+
+.chrome {
+  display: grid;
+  grid-template-columns: 6fr 1fr;
+  margin: 1px 8px;
+  height: 10vh;
+  background-color: #fdf8f6;
+  padding: 0 0 2px 10px;
+}
+
+.chrome img {
+  width: 80px;
+  height: 100%;
+}
+
+.chrome > div {
+  width: 90%;
+}
+
+.chrome-links {
+  margin: 25px 10px;
+  width: 85%;
+}
+
+.deals {
+  display: flex;
+  justify-content: space-between;
+}
+
+footer {
+  position: relative;
+  background: linear-gradient(rgb(36, 33, 33) 50%, rgb(12, 12, 12) 50%);
+  height: 40vh;
+}
+
+.social {
+  padding: 30px 10px;
+  align-content: center;
+  width: 100%;
+}
+
+.social ul {
+  display: flex;
+  font-size: 1.3rem;
+  justify-content: center;
+  list-style-type: none;
+  width: 50%;
+  margin: auto;
+}
+
+.social ul li {
+  border: 1px solid rgb(36, 33, 33);
+  background: #333;
+  height: 40px;
+  width: 40px;
+  border-radius: 50%;
+  padding: 8px;
+  text-align: center;
+}
+
+.footer-links {
+  padding: 5px 5px;
+  align-content: center;
+}
+
+.footer-links ul {
+  display: flex;
+  font-size: 0.7rem;
+  justify-content: center;
+  list-style-type: none;
+}
+
+.footer-links ul li {
+  padding-left: 7px;
+  text-align: center;
+}
+
+.bottom-footer {
+  margin: 6em auto 0 auto;
+  text-align: center;
+  width: 60%;
+  color: #fdf8f6;
+}
+
+.date {
+  display: none;
+}
+
+.hidden,
+.desktop {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  .nav-info {
+    display: flex;
+    background-color: black;
+  }
+
+  .nav-info a {
+    font-size: 1rem;
+    color: gray;
+  }
+
+  .c-menu {
+    display: flex;
+    justify-content: space-between;
+    background-color: white;
+  }
+
+  .c-menu a {
+    font-size: 1rem;
+    color: gray;
+  }
+
+  .fa-bars {
+    display: none;
+  }
+
+  .right-nav {
+    width: 100%;
+  }
+
+  .right-nav ul li {
+    border: none;
+  }
+
+  .search {
+    border-left: 1px solid rgb(212, 212, 212);
+    padding-left: 15px;
+  }
+
+  .hidden {
+    display: flex;
+  }
+
+  nav {
+    height: 90px;
+  }
+
+  .logo img {
+    width: 110px;
+    height: 90px;
+  }
+
+  .news,
+  .news2,
+  .news3 {
+    height: 100%;
+    width: 100%;
+  }
+
+  .news {
+    grid-area: news;
+  }
+
+  .news2 {
+    grid-area: news2;
+  }
+
+  .news3 {
+    grid-area: news3;
+  }
+
+  .news-holder {
+    display: grid;
+    grid-template-areas:
+      'news news news2'
+      'news news news3';
```

Comment:
Please use this  [article](https://developer.mozilla.org/es/docs/Web/CSS/grid-template-areas)  to determine the proper way to set the grid